app/tests/unit/containers/Admin/NaicsCodes/NaicsCodeTable.test.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+import {NaicsCodeTableContainer} from 'containers/Admin/NaicsCode/NaicsCodeTable';
+
+describe('NaicsCodeTable', () => {
+  it('should match the snapshot with the NaicsCodeTable container', async () => {
+    const renderer = shallow(
+      <NaicsCodeTableContainer
+        relay={null}
+        query={{
+          ' $refType': 'NaicsCodeTable_query',
+          allNaicsCodes: {
+            __id: 'abc',
+            edges: [
+              {
+                node: {
+                  ' $fragmentRefs': {
+                    NaicsCodeTableRow_naicsCode: true
+                  }
+                }
+              }
+            ]
+          }
+        }}
+      />
+    );
+    expect(renderer).toMatchSnapshot();
+  });
+  it('should render the NaicsCodeTableRow container', async () => {
+    const renderer = shallow(
+      <NaicsCodeTableContainer
+        relay={null}
+        query={{
+          ' $refType': 'NaicsCodeTable_query',
+          allNaicsCodes: {
+            __id: 'abc',
+            edges: [
+              {
+                node: {
+                  ' $fragmentRefs': {
+                    NaicsCodeTableRow_naicsCode: true
+                  }
+                }
+              }
+            ]
+          }
+        }}
+      />
+    );
+    expect(renderer.exists('Relay(NaicsCodeTableRowContainer)')).toBe(true);
+  });
+
+  it('should render the CreateNaicsCode modal component', async () => {
```

Comment:
Add a test that ensures that the modal 'open' prop changes when clicking on the "New NAICS code" button
and another one that ensures that `createNaicsCodeMutation is called when submitting the form